README.md

```diff
@@ -1,7 +1,167 @@
+</p>
 
-# 🤝 Contributing
+<h3 align="center">ft_transcendence</h3>
+
+<div align="center">
+
+[![Status](https://img.shields.io/badge/status-active-success.svg)]()
+[![GitHub Issues](https://img.shields.io/github/issues/42agv/ft_transcendence.svg)](https://github.com/42agv/ft_transcendence/issues)
+[![GitHub Pull Requests](https://img.shields.io/github/issues-pr/42agv/ft_transcendence.svg)](https://github.com/42agv/ft_transcendence/pulls)
+
+</div>
+
+---
+
+<p align="center"> 🏓 Play pong online with your friends in real time
+  <br>
+</p>
+
+## 📝 Table of Contents
+
+- [About](#about)
+- [Getting Started](#getting_started)
+- [Deployment](#deployment)
+- [Usage](#usage)
+- [Built Using](#built_using)
+- [Contributing](#contributing)
+- [Acknowledgments](#acknowledgement)
+
+## 🧐 About <a name = "about"></a>
+
+The primary purpose of this website is to play pong against other players and show everyone how good you are!
+
+## 🏁 Getting Started <a name = "getting_started"></a>
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See [deployment](#deployment) for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+What things do you need to install the software, and how to install them.
+
+[git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+[docker compose](https://docs.docker.com/compose/install/)
+
+[npm](https://docs.npmjs.com/cli/v7/configuring-npm/install)
+
+[node](https://nodejs.dev/learn/how-to-install-nodejs)
+
+### Installing
+
+A step-by-step series of examples that tell you how to get a development environment running.
+
+Clone the GitHub repository
+
+```
+git clone https://github.com/42AGV/ft_transcendence.git
+```
+
+Change the current working directory to `transcendence-app`
+
+```
+cd ft_transcendence/transcendence-app
+```
+
+Copy the `.env.sample` to `.env`
+
+```
+cp .env.sample .env
+```
+
+Register a new OAuth application on the [42 intranet](https://profile.intra.42.fr/oauth/applications/new)
+
+Set the Redirect URIs
+
+```
+http://127.0.0.1:3000/api/v1/auth/login
+http://127.0.0.1/api/v1/auth/login
+```
+
+Open the `.env` file with your favorite text editor and replace the ID and SECRET with the ones from the OAuth application you create in the previous step
+
+```
+FORTYTWO_APP_ID=42-intra-app-id
+FORTYTWO_APP_SECRET=42-intra-app-secret
+```
+
+Generate two UUIDs using for example the `uuidgen` command
+
+Replace the SESSION_SECRET and the MEMCACHED_SECRET with the UUIDs from the previous step
+
+```
+SESSION_SECRET=your-UUID-here
+MEMCACHED_SECRET=your-UUID-here
+```
+
+Change the working directory to the root directory
+
+```
+cd ..
+```
+
+Start the application using `Docker`
+
+```
+docker compose up -d --build
+```
+
+You can now access the web application at http://localhost:3000 and the Swagger documentation for the API at http://localhost:3000/api
```

Comment:
generally, when I run `docker-compose up --build`, I can also see the website at
`ft_transcendence-webapp-1             |   Local:            http://localhost:80` (if I disable the default web server) or
`ft_transcendence-webapp-1             |   On Your Network:  http://172.xx.xx.xx:80`. Maybe it's worth mentioning? It is prompted. Maybe not worth it.

Reply:
I think accessing the web app using the Docker IP is an advanced use case and bypasses the reverse proxy.
I would not add it to the documentation because while it is possible, the IP is assigned automatically by Docker.

Reply:
On second thought, it might be useful for debugging.
I've added a note on how to access a service using the Docker container IP